src/fetch-native-lwt/fetch_native_lwt.re

```diff
@@ -0,0 +1,82 @@
+module IO = {
+  module Response = {
+    module Status = {
+      include Fetch_core.Response.Status;
+    };
+
+    module Headers = {
+      include Fetch_core.Headers;
+    };
+
+    module Body = {
+      type t = Morph.Response.body;
+
+      let toString = body => {
+        switch (body) {
+        | `String(body) => body
+        | `Stream(chars) => Lwt_stream.to_string(chars) |> Lwt_main.run
```

Comment:
This is blocking, can you make the `toString` return a `Lwt.t(string)` instead?

Actually, currently we never return a `Stream from the client so you can no-op that part if you want to, but this will probably change with H2 support.